ibacm/ibacm.service.in

```diff
@@ -1,12 +1,12 @@
 [Unit]
 Description=Starts the InfiniBand Address Cache Manager daemon
-Documentation=man:ibacm file:/etc/rdma/ibacm_opts.cfg
+Documentation=man:ibacm file:@CMAKE_INSTALL_SYSCONFDIR@/rdma/ibacm_opts.cfg
 Requires=rdma.service
 After=rdma.service opensm.service
```

Comment:
We don't have rdma.service in rdma-core, that is a redhat specific target.. It should be stripped out before moving..